// umt/module/Math/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.toCelsius = exports.toKelvin = exports.multiples = exports.factorize = exports.standardDeviation = exports.average = exports.gcd = exports.random = exports.calculator = exports.primeFactorization = exports.radToDeg = exports.degToRad = exports.toBinary = exports.isPrimeNumber = exports.division = exports.addition = exports.multiplication = exports.subtract = exports.getDecimalLength = exports.valueSwap = exports.isNumber = exports.isDouble = exports.min = exports.max = exports.repeatedTrial = exports.reduce = exports.quotient = exports.nPr = exports.nHr = exports.nCr = exports.lcm = exports.factorial = void 0;
const addition_1 = __importDefault(require("./addition"));
exports.addition = addition_1.default;
const average_1 = __importDefault(require("./average"));
exports.average = average_1.default;
const calculator_1 = __importDefault(require("./calculator"));
exports.calculator = calculator_1.default;
const degToRad_1 = __importDefault(require("./degToRad"));
exports.degToRad = degToRad_1.default;
const division_1 = __importDefault(require("./division"));
exports.division = division_1.default;
const gcd_1 = __importDefault(require("./gcd"));
exports.gcd = gcd_1.default;
const factorial_1 = __importDefault(require("./factorial"));
exports.factorial = factorial_1.default;
const factorize_1 = __importDefault(require("./factorize"));
exports.factorize = factorize_1.default;
const getDecimalLength_1 = __importDefault(require("./getDecimalLength"));
exports.getDecimalLength = getDecimalLength_1.default;
const isDouble_1 = __importDefault(require("./isDouble"));
exports.isDouble = isDouble_1.default;
const isNumber_1 = __importDefault(require("./isNumber"));
exports.isNumber = isNumber_1.default;
const isPrimeNumber_1 = __importDefault(require("./isPrimeNumber"));
exports.isPrimeNumber = isPrimeNumber_1.default;
const lcm_1 = __importDefault(require("./lcm"));
exports.lcm = lcm_1.default;
const max_1 = __importDefault(require("./max"));
exports.max = max_1.default;
const min_1 = __importDefault(require("./min"));
exports.min = min_1.default;
const multiples_1 = __importDefault(require("./multiples"));
exports.multiples = multiples_1.default;
const multiplication_1 = __importDefault(require("./multiplication"));
exports.multiplication = multiplication_1.default;
const nCr_1 = __importDefault(require("./nCr"));
exports.nCr = nCr_1.default;
const nHr_1 = __importDefault(require("./nHr"));
exports.nHr = nHr_1.default;
const nPr_1 = __importDefault(require("./nPr"));
exports.nPr = nPr_1.default;
const primeFactorization_1 = __importDefault(require("./primeFactorization"));
exports.primeFactorization = primeFactorization_1.default;
const quotient_1 = __importDefault(require("./quotient"));
exports.quotient = quotient_1.default;
const radToDeg_1 = __importDefault(require("./radToDeg"));
exports.radToDeg = radToDeg_1.default;
const random_1 = __importDefault(require("./random"));
exports.random = random_1.default;
const reduce_1 = __importDefault(require("./reduce"));
exports.reduce = reduce_1.default;
const repeatedTrial_1 = __importDefault(require("./repeatedTrial"));
exports.repeatedTrial = repeatedTrial_1.default;
const standardDeviation_1 = __importDefault(require("./standardDeviation"));
exports.standardDeviation = standardDeviation_1.default;
const subtract_1 = __importDefault(require("./subtract"));
exports.subtract = subtract_1.default;
const toBinary_1 = __importDefault(require("./toBinary"));
exports.toBinary = toBinary_1.default;
const toCelsius_1 = __importDefault(require("./toCelsius"));
exports.toCelsius = toCelsius_1.default;
const toKelvin_1 = __importDefault(require("./toKelvin"));
exports.toKelvin = toKelvin_1.default;
const valueSwap_1 = __importDefault(require("./valueSwap"));
exports.valueSwap = valueSwap_1.default;
